internals/secrethub/credential_list_test.go

```diff
@@ -0,0 +1,82 @@
+package secrethub
+
+import (
+	"testing"
+
+	"github.com/secrethub/secrethub-cli/internals/cli/ui/fakeui"
+	"github.com/secrethub/secrethub-go/internals/api"
+	"github.com/secrethub/secrethub-go/internals/assert"
+	"github.com/secrethub/secrethub-go/internals/errio"
+	"github.com/secrethub/secrethub-go/pkg/secrethub"
+	"github.com/secrethub/secrethub-go/pkg/secrethub/fakeclient"
+)
+
+func TestCredentialListCommand_Run(t *testing.T) {
+	testErr := errio.Namespace("test").Code("test").Error("test error")
+
+	cases := map[string]struct {
+		cmd               CredentialListCommand
+		credentialService fakeclient.CredentialService
+		newClientErr      error
+		out               string
+		err               error
+	}{
+		"success list credentials": {
```

Comment:
Since we're testing the credential list command, this test case can be named just `success`

Reply:
Nice suggestion. Done